jmix-translations/content/io/jmix/flowui/messages_ru.properties

```diff
@@ -336,4 +336,57 @@ io.jmix.flowui.component.genericfilter.configuration/filterConfigurationDetail.g
 
 #GenericFilterEditAction
 io.jmix.flowui.action.genericfilter/genericFilterEditAction.filterConfigurationDetail.title = Конфигурация
-io.jmix.flowui.action.genericfilter/genericFilterEditAction.filterConfigurationDetail.groupConditionTitle = Корневое групповое условие
\ No newline at end of file
+io.jmix.flowui.action.genericfilter/genericFilterEditAction.filterConfigurationDetail.groupConditionTitle = Корневое групповое условие
+
+#JMX console
+
+io.jmix.jmxconsole/title=Консоль JMX
+
+io.jmix.jmxconsole.model/ManagedBeanInfo.domain=Домен
+io.jmix.jmxconsole.model/ManagedBeanInfo.className=Название класса
+io.jmix.jmxconsole.model/ManagedBeanInfo.objectName=Название объекта
+io.jmix.jmxconsole.model/ManagedBeanInfo.description=Описание
+
+io.jmix.jmxconsole.model/ManagedBeanAttribute.valueString=Значение
+io.jmix.jmxconsole.model/ManagedBeanAttribute.readableWriteable=Доступ
+io.jmix.jmxconsole.model/ManagedBeanAttribute.type=Тип
+io.jmix.jmxconsole.model/ManagedBeanAttribute.description=Описание
+io.jmix.jmxconsole.model/ManagedBeanAttribute.name=Имя
+
+io.jmix.jmxconsole/caption=Консоль JMX
+io.jmix.jmxconsole/action.inspect=Просмотреть MBean
+io.jmix.jmxconsole/searchByObjectName=Поиск по названию объекта
+
+io.jmix.jmxconsole/mbean.title=Просмотр MBean объекта "%s"
+io.jmix.jmxconsole/mbean.className=Класс: %s
+io.jmix.jmxconsole/mbean.description=Описание: %s
+io.jmix.jmxconsole/mbean.name=Объект: %s
+io.jmix.jmxconsole/mbean.operations=Операции
+io.jmix.jmxconsole/mbean.operations.none=Нет
+io.jmix.jmxconsole/mbeanDetailView.title=Редактирование Mbean
+io.jmix.jmxconsole/mbean.attributes=Атрибуты
+io.jmix.jmxconsole/attribute.edit=Изменить значение
+io.jmix.jmxconsole/close=Закрыть
+
+io.jmix.jmxconsole/editAttribute.caption.format=Изменить значение атрибута "%s"
+io.jmix.jmxconsole/editAttribute.value=Значение
+io.jmix.jmxconsole/inspectAction.text=Просмотреть
+io.jmix.jmxconsole/editAttribute.exception=Исключение во время сохранения атрибута: "%s"
+io.jmix.jmxconsole/editAttribute.conversionError=Не удалось изменить атрибут данного типа
+io.jmix.jmxconsole/editAttribute.array.btnAdd=Добавить строку к масиву
+io.jmix.jmxconsole/editAttribute.array.btnRemove=Удалить строку из масива
+
+io.jmix.jmxconsole/operationResult.caption=Результат операции
```

Comment:
What about this key? Do you use it or `mbeanOperationResult.title` its replacement? Why `mbeanOperationResult.title` isn't translated to the other languages?

Check your localization keys and their usage